.github/workflows/build-natives.yml

```diff
@@ -0,0 +1,118 @@
+name: Build native dependencies
+
+env:
+    OPENALSOFT_VERSION: 1.20.1
```

Comment:
Can we pass that in as parameter? Does that make sense? (I assume this is something that might change.)

Reply:
Ok.